.ci/docker/common/install_conda.sh

```diff
@@ -9,6 +9,18 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   MAJOR_PYTHON_VERSION=$(echo "$ANACONDA_PYTHON_VERSION" | cut -d . -f 1)
   MINOR_PYTHON_VERSION=$(echo "$ANACONDA_PYTHON_VERSION" | cut -d . -f 2)
 
+if [[ $(uname -m) == "aarch64" ]]; then
+  BASE_URL="https://github.com/conda-forge/miniforge/releases/latest/download"
```

Comment:
Why prefer miniforge rather than miniconda, which is more stable?

Reply:
I see several packages don't exist on miniconda channel for aarch64. And also, we've been using the miniforge in the CD scripts.https://github.com/pytorch/builder/blob/main/aarch64_linux/aarch64_ci_setup.sh